### DISCUSSION TITLE
[IMPORVE]异步回复公众号消息

### DISCUSSION THREAD

**Discussion by tsfans** [pwh-pwh/aiwechat-vercel#10] [Ideas]
Vercel freeplan超时时间太短，很多对话都得不到响应，考虑通过异步回复方式提升体验

**Comment by pwh-pwh** [comment]
这个应该和异步没有关系，占用最长的时间就是在获取回答上面，这个时间减少不了。使用国内的ai可以把serveless的区域改到亚洲改善。
不然就只能先回复个稍后再试，然后把答案存起来，下次提问直接返回（个人不是很喜欢这样子）

**Reply by tsfans** [reply]
微信好像是可以通过客户消息来异步推送回复的，这样就不用受限于function的超时时间

**Reply by pwh-pwh** [reply]
这个function的时间计算的是接口到达直至响应的时间吧，也就是从后台接收到请求，到返回信息给微信后台的时间。所以对于异步推送也是一样的效果
